lib/content/mixin.js

```diff
@@ -8,6 +8,7 @@ const interactionEvents = [
   'updateProgress',
   'updateContentState',
   'startTracking',
+  'navigateto',
```

Comment:
You called your hashi event `navigateto`, but the event you are emitting from the content renderer (which is relevant to this) was called `navigateToRegularContext` https://github.com/learningequality/kolibri/pull/8134/files#diff-cf6ea11127dacf9dddc24383c075d4e76547563841df759b5bbf0962c9293719R127

I certainly prefer `navigateTo` but perhaps with camelCase to be consistent.